src/nginxconfig/templates/footer.vue

```diff
@@ -68,5 +68,14 @@ THE SOFTWARE.
         components: {
             ExternalLink,
         },
+        methods: {
+            handleScrollToTop: () => {
+                window.scrollTo({top: 0});
+
+                const columns = document.querySelectorAll('.column-scroll-y');
+
+                columns.forEach(column => column.scrollTo({top: 0}));
```

Comment:
```suggestion
                window.scrollTo({ top: 0 });
                document.querySelectorAll('.column-scroll-y')
                  .forEach(column => column.scrollTo({ top: 0 }));
```
Also a nitpick, formatting + removing the redundant const (lmk if eslint disagrees with this)

Reply:
Ya, you are right. I have removed redundant cosnt.